chevah/compat
# [#2141] Remove home folders only when justified.

## Problem

We need to remove home folders for test users only if the test and/or testsuite generated the creation.
## Changes
- Tests for `getHomeFolder` on Windows are removing the created folder on teardown. The testsuite is removing the home folders for all users created with `create_local_profile` flag enabled.
- Test for `setOwner_ok` will check for folder too.
## How to test

Please check that the changes look ok.

I've tried to use a manually set the home folder to the temporary folder. Even thougsh I've manually set a value for it the folder was still created in `C:\Users`. We might use a different API for production code. I will check it first thing tomorrow.

Please take a quick look to see if we can leave with this if there is no easy way to do it. Thanks.
